### PR TITLE
Embed a file

### PR DESCRIPTION
**Pull request by KJEZ35**
Hopefully I get this right!

### CODE REVIEW

File: 2020-09-09/KJEZ35-logical-turtle.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+author: KJEZ35
+layout: post
+title: "embed a tutrtle"
```
Comment (brianna-stedman) [inline 2020-09-09/KJEZ35-logical-turtle.md]: "turtle" :)

**Review by brianna-stedman — CHANGES_REQUESTED** [review]
Looks great! 

I would adjust the file name to make sure it's going where it's supposed to go, and to keep the same naming convention for posts. Should look like:

_posts/KJEZ35/2020-09-09-KJEZ35-logical-turtle.md

File: 2020-09-09/KJEZ35-logical-turtle.md (hunk 1 above)
Comment (KJEZ35) [reply 2020-09-09/KJEZ35-logical-turtle.md]: Thank you!

Comment (brianna-stedman) [reply 2020-09-09/KJEZ35-logical-turtle.md]: also the file name needs to be changed as I mentioned above! other than that, looks good! can't wait to see your trinket @KJEZ35

**Review by brianna-stedman — CHANGES_REQUESTED** [review]
one more change, because I'm not sure whether it will affect - but make sure your title is 

2020-09-09-KJEZ35-logical-turtle.md

and not 

2020-09-09/KJEZ35-logical-turtle.md

I don't know if the extra forward slash will affect anything, but just to be sure -

File: 2020-09-09/KJEZ35-logical-turtle.md (hunk 1 above)
Comment (KJEZ35) [reply 2020-09-09/KJEZ35-logical-turtle.md]: File name was changed, Thank you!

**Review by brianna-stedman — APPROVED** [review]
looks perfect

**Comment by brianna-stedman** [comment]
@KJEZ35 I merged it!

**Comment by KJEZ35** [comment]
So I went back and edited my class activity so to include all the bench marks. I was having issues finding a way to edit my pull request to add it. Until I can figure it out, I will add it here: 


<iframe src="https://trinket.io/embed/python/5a0a4b15ca" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>

**Pull request merged**